cdbot/cogs/fun.py

```diff
@@ -577,6 +582,24 @@ async def baldj(self, ctx: Context, *, text: str):
         """
         await self.create_text_image(ctx, "AgentJBadHairDay", text)
 
+    # Polls
+    @command()
+    async def suggest(self, ctx: Context, *, poll_question: str):
+        confirm_msg = Embed(
+            description=f":white_check_mark: Your suggestion has been sent to <#{POLL_CHANNEL_ID}> to be voted on.",
+            color=0x6ABE6C)
+        confirm_msg.set_author(name=ctx.message.author.name, icon_url=ctx.message.author.avatar_url)
+        await ctx.send(embed=confirm_msg)
+
+        poll_webhook = await self.bot.fetch_webhook(POLL_WEBHOOK_ID)
```

Comment:
Unused assignment

Reply:
I don't think it is...

Reply:
`poll_webhook` is used on line 607, @thebeanogamer